modules/ROOT/pages/install-validate.adoc

```diff
@@ -6,13 +6,13 @@ endif::[]
 
 To verify your installation and configuration:
 
-1. Verify that your Anypoint organization has the API Community Manager (ACM) entitlement. Log in to your Anypoint Platform organization and navigate in your browser to `+https://anypoint.mulesoft.com/accounts/api/profile+`. Check that under the *Entitlements* property the `apiCommunityManager` value is `true`.
+1. Verify that your Anypoint organization has the Anypoint API Community Manager entitlement. Log in to your Anypoint Platform organization and navigate in your browser to `+https://anypoint.mulesoft.com/accounts/api/profile+`. Check that under the *Entitlements* property the `apiCommunityManager` value is `true`.
```

Comment:
Why aren't these formatted as steps?
Also, ditto one sentence per step.
You can use substeps (formatted ..).

Reply:
```suggestion
. Verify that your Anypoint organization has the Anypoint API Community Manager entitlement. 
.. Log in to your Anypoint Platform organization and navigate in your browser to `+https://anypoint.mulesoft.com/accounts/api/profile+`. 
.. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
```